### ISSUE TITLE
Modify Description on Link on Home Page in Commons Participant Section

### ISSUE THREAD

**Issue by dmueller2001** [openshift/commons.openshift.org#455]
Current String: "View the Full Participant List (265 members)"

New String: "View the Full Participant List (275+ members)"
![screen shot 2017-06-21 at 1 11 24 pm](https://user-images.githubusercontent.com/4029488/27404506-2b161762-5683-11e7-8038-855858141679.png)

**Comment by jiri-fiala** [comment]
@dmueller2001, the member count is [automatically generated](https://github.com/openshift/commons.openshift.org/blob/master/source/index.html.erb#L206) based on actual number of items in [the participant list](https://github.com/openshift/commons.openshift.org/blob/master/data/participants.yml).
If the list ^ is not up to date, we can either update it with the missing members, or maybe reword the link from _"View the Full Participant List (xxx members)"_ to a static _"View the Participant List"_ (removing the number and the "full" mention)? I would like to avoid putting there the "275+" number until the list has at least 275 members published.

**Comment by jiri-fiala** [comment]
I'm closing this based on discussion with Diane. There are some participants who do not generally allow their names to be referenced, so we can't add them to [the list](https://github.com/openshift/commons.openshift.org/blob/master/data/participants.yml) at the moment.
Only those included in the list are to be counted in the link for now -> no change.
